### PR TITLE
Update Go modules for 2.26

### PR DESCRIPTION
**Pull request by SuperQ**
Bump all Go modules to the latest upstream.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Wow, thanks for help on this!

File: cmd/promtool/main.go [hunk 1]
```diff
@@ -640,7 +640,7 @@ func QueryLabels(url *url.URL, name string, start, end string, p printer) int {
 	// Run query against client.
 	api := v1.NewAPI(c)
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, warn, err := api.LabelValues(ctx, name, stime, etime)
+	val, warn, err := api.LabelValues(ctx, name, []string{}, stime, etime)
```
Comment (bwplotka) [inline cmd/promtool/main.go:643]: Is that required? 🤔

**Review by bwplotka — APPROVED** [review]
LGTM, just one question. Thanks!

File: cmd/promtool/main.go (hunk 1 above)
Comment (bwplotka) [reply cmd/promtool/main.go:643]: > LabelValues now requires a list of string matchers.

Got my answer, looks good then, but `nil` would do too.

Comment (SuperQ) [reply cmd/promtool/main.go:643]: https://github.com/prometheus/client_golang/pull/828

**Pull request merged**